tests/dummy/app/styles/app.scss

```diff
@@ -1,22 +1,145 @@
 @import "bower_components/uniq/src/styles/_main";
+@import url('https://fonts.googleapis.com/css?family=Open+Sans|Pacifico');
 
 html,
 body {
   width: 100%;
-  min-height: 100%;
-  font-family: 'Roboto', sans-serif;
-  font-size: 16px;
-  margin: 10px 0;
+  height: 100vh;
+  font-family: 'Open Sans', sans-serif;
+  font-size: em(16px);
   -webkit-font-smoothing: antialiased;
 }
 
+.uni-title,
+.uni-subtitle {
+  font-family: 'Open Sans', sans-serif;
+}
+
+.demo-app__banner {
+  width: 100%;
+  height: 156px;
+  padding: 32px 0 24px;
+  background-color: color("primary");
+  text-align: center;
+  position: relative;
+  display: flex;
+  z-index: 1;
+
+  transition: height .2s ease-out;
+
+  .uni-title {
+    font-size: em(52px);
+    font-family: 'Pacifico', cursive;
+    color: #fff;
+    background-color: color("primary");
+    display: flex;
+    flex-direction: column;
+
+    .uni-subtitle {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      margin-top: 8px;
+      color: #fff;
+      font-size: em(16px);
+
+      svg {
+        height: 24px;
+        width: 100px;
+        margin-left: 8px;
+      }
+    }
+  }
+
+  &__current-version {
+    color: #fff;
+    text-align: right;
+    font-size: em(14px);
+    position: absolute;
+    right: 16px;
+    bottom: 16px;
+
+    span {
+      font-weight: 700;
```

Comment:
should we be using our font here? and therefore no font-weights?

Reply:
We don't have our font in ember-cli-uniq as it is paid, i think

Reply:
😿